docker/l2s/src/config/config.ts

```diff
@@ -38,7 +38,8 @@ export function getConfig(): Config {
     dbUser: getEnv("POSTGRES_USER", "postgres"),
     dbPassword: getEnv("POSTGRES_PASSWORD", "postgres"),
     dbDatabase: getEnv("POSTGRES_DATABASE", ""),
-    dbSslEnabled: getEnv("POSTGRES_SSL_ENABLED", "true") === "true",
+    rejectUnauthorized:
+      getEnv("POSTGRES_REJECT_UNAUTHORIZED", "false") === "true",
```

Comment:
không parse true/false string đc à

Reply:
chỗ này em chưa rõ lắm anh, phần parse đó em làm theo: [Stackoverflow Reference](https://stackoverflow.com/questions/263965/how-can-i-convert-a-string-to-boolean-in-javascript)